packages/kit/src/exports/public.d.ts

```diff
@@ -473,15 +473,18 @@ export interface KitConfig {
 		 */
 		base?: '' | `/${string}`;
 		/**
-		 * Whether to use relative asset paths. By default, if `paths.assets` is not external, SvelteKit will replace `%sveltekit.assets%` with a relative path and use relative paths to reference build artifacts, but `base` and `assets` imported from `$app/paths` will be as specified in your config.
+		 * Whether to use relative asset paths. `true` by default in order to support things like the Internet Archive. `false` may have slight performance advantages and may be preferrable for internal applications.
```

Comment:
I don't think we need this. 'may have slight performance advantages' seems like bollocks to me (why would it?) and will encourage people to do the less resilient thing

```suggestion
		 * Whether to use relative asset paths.
```

Reply:
>  'may have slight performance advantages' seems like bollocks to me (why would it?)

There are numerous reasons why it does. Perhaps we can address those though, so I'll remove this for now